scripts/metadata_comparison/test/resources/comparer/valid_comparison_result.csv

```diff
@@ -0,0 +1,3 @@
+,version,calls.root_wf.sub_wf.shard_0.call_foo.shard_0.overallRuntimeSeconds,calls.root_wf.sub_wf.shard_1.call_bar.shard_2.overallRuntimeSeconds,calls.root_wf.baz.overallRuntimeSeconds
```

Comment:
```suggestion
workflow,digester version,calls.root_wf.sub_wf.shard_0.call_foo.shard_0.overallRuntimeSeconds,calls.root_wf.sub_wf.shard_1.call_bar.shard_2.overallRuntimeSeconds,calls.root_wf.baz.overallRuntimeSeconds
```

Reply:
Not sure I understand this

Reply:
It'd be nice if there was a title for the first column (suggesting `workflow` here) and if the second column told us what version it was (suggesting `digester version` here)